graphql-rails-generators.gemspec

```diff
@@ -12,4 +12,5 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.homepage    = 'https://github.com/ajsharp/graphql-rails-generators'
   s.license     = 'MIT'
+  s.add_runtime_dependency 'search_object_graphql'
```

Comment:
Let's remove `search_object_graphql` as a runtime dependency, since it's only required if you're using this particular generator.

Reply:
@jdpaterson I think we're good once this change is made.

Reply:
@ajsharp changes have been made. Thanks!